release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -2919,6 +2919,12 @@ It is anticipated that an upcoming z-stream release will include a fix for this
 +
 To work around this issue: If your control plane nodes are schedulable, and the pods can run on worker nodes, use `nodeSelector` or Affinity to schedule the pod in worker nodes.(link:https://issues.redhat.com/browse/OCPBUGS-18581[*OCPBUGS-18581*])
 
+* Currently, the update to HAProxy version 2.6.13 in {product-title} {product-version} causes an increase in P99 latency for re-encrypt traffic. This is observed when the volume of ingress traffic puts the HAProxy component of the `IngressController` custom resource (CR) under a considerable load. The latency increase does not affect overall throughput, which remains consistent.
++
+The default `IngressController` CR is configured with 4 HAProxy threads. If you experience elevated P99 latencies during high ingress traffic conditions, specifically with re-encrypt traffic, it's recommended to increase the number of HAProxy threads to reduce latency. (link:https://issues.redhat.com/browse/OCPBUGS-18936[*OCPBUGS-18936*])
+
+* For {sno-caps} on {product-version} and Google Cloud Platform, there is a known issue with the Cluster Network Operator getting stuck on a crash loop because the cloud network config controller recieves the wrong IP address from the name server when trying to reach the API server. Currently, there is no workaround for this issue. (link:https://issues.redhat.com/browse/OCPBUGS-20554[*OCPBUGS-20554*])
```

Comment:
Not 'Cluster Network Operator ' on a crash loop, but CNCC(cloud network config controller) on a crash loop.

Reply:
I'd say something along the lines of:

`* For {sno-caps} on {product-version} and Google Cloud Platform, there is a known issue with the Cloud Network Config Controller being in CrashLoopBackOff when at initialization time it tries to reach the Azure internal load balancer address: the resulting hairpin traffic is not correctly prevented in OVN shared gateway mode on Google Cloud Platform and gets dropped. Cluster Network Operator will show Progressing=true in such case. The fix to this bug is currently being worked on and will be available in a future {product-version} z-stream release. (link:https://issues.redhat.com/browse/OCPBUGS-20554[*OCPBUGS-20554*])`